scalafmt-tests/src/test/resources/test/arityThreshold.source

```diff
@@ -10,6 +10,7 @@ case class VeryLongNames(xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx: Str
 object A {
   def foo(x: String, y: String)
   def hello(how: String)(are: String)(you: String) = how + are + you
+  def other(a: String, b: String)(c: String) = a + b + c
```

Comment:
Pro tip 😉 it's good to make this a standalone test case. You can rename arityThreshold.source into arityThreshould.stat and then write unit tests like

```scala
<<< Issue 101
def other(a: String, b: String)(c: String) = a + b + c 
>>>
def other(
  a: String,
  b: String
)(c: String) = a + b + c 
```